js/samples/docs-menu-generation/README.md

```diff
@@ -0,0 +1,30 @@
+## Build it
+
+```
+pnpm build
+```
+
+or if you need to, build everything:
```

Comment:
Maybe we can set this up as a prerequisite step (build the entire repo first) that we note can be skipped if already done. otherwise, how does one answer the question "build if you need to?"